### ISSUE TITLE
Interfacing LCD with Beaglebone Black Wireless

### ISSUE THREAD

**Issue by joyalraju** [joyalraju/Capstone-Project-Summer-2020#11]

**Comment by mikeshams** [comment]
1. What issues did you or others might face with connecting the LCD Display to the BBB Wireless in terms of Hardware or Software?
2. How did you share your screen on Adobe Connect while using Linux?
3. Did you use dbt files? Why not?

**Comment by sruthykrishnan** [comment]
1. Data is send bit by bit to LCD which makes it difficult while coding. Also we need to send control signals along with the data which should be send in bit by bit format, which makes it difficult while coding. While doing hardware connection, normal connection with LCD was a challenge as it required bit by bit data. So I2C communication is one of the easy ways for interfacing. These are the challenges I faced during Interfacing LCD with BBB Wireless
2. I used Putty to share my terminal window for Linux through Adobe Connect. Putty is a free and open source terminal emulator which supports several network protocols including SSH. It can provide secure remote shell access to Linux systems. For that I downloaded Putty for windows.Then I selected the ssh connection and set the host address to 192.168.7.2. Then it will open the Linux terminal window.
3. In this program I am not using dtb. dtb is the device tree binary which is used to pass hardware information about the board to the Linux Kernel. We are interfacing separately with separate C code which is in dts(device tree source code). After doing all the interfacing, for the main program we will be combining it all together and will be using dtb.
